services/src/modules/directives/rest/datasource.ts

```diff
@@ -86,10 +89,16 @@ export class RESTDirectiveDataSource extends RESTDataSource<RequestContext> {
     args: GraphQLArguments,
     info: GraphQLResolveInfo
   ) {
-    if (!kvs || kvs.length == 0) return;
+    let validQueryParams = true;
+
+    if (!kvs || kvs.length == 0) return validQueryParams;
 
     for (const kv of kvs) {
       const value = inject(kv.value, parent, args, this.context, info);
+      if (!value && kv.required) {
+        validQueryParams = false;
+        break;
+      }
       if (!value) {
```

Comment:
We already have bug here :-)
```suggestion
      if (value === undefined) {
```

Reply:
Same comment as above. I don't think this was an bug.